worker/address-stats.js

```diff
@@ -0,0 +1,71 @@
+const moment = require('moment');
+const utils = require('web3-utils');
+const { getAddress } = require('../lib/address');
+const { useAddressStatsWorker } = require('../lib/agenda');
+const { useDb } = require('../lib/db');
+
+const agenda = useAddressStatsWorker();
+
+agenda.define('AddressStatsWorker', async (job) => {
+  try {
+    const { id } = job.attrs.data;
+    if (!utils.isHexStrict(id)) {
+      console.log(`Skipping address stats for invalid ${id}`);
+      return;
+    }
+
+    console.log(`Processing address stats for ${id}`);
+    const { Address, Block, Tx } = await useDb();
+
+    const now = moment();
+    let address = await getAddress(id);
+    if (!address) {
+      console.log(`Creating new address stats for ${id}`);
+      address = new Address({ _id: id, updatedAt: now.toDate(), createdAt: now.toDate() });
```

Comment:
You can remove `updatedAt` and `createdAt` because the Mongo library populates them automatically.

Reply:
done